website/public/js/controllers/inviteToGroupCtrl.js

```diff
@@ -26,6 +26,11 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', 'User', 'Groups', 'injectedG
       _inviteByMethod(inviteMethod);
     };
 
+    $scope.getMemberAndInviteCount = function(group) {
+
+      return group.memberCount + group.invites.length;
```

Comment:
True, but why does the error reference line 9?  A similar line to this was on line 9 in a previous commit and it produced the same exact error.  But now that line is gone.